docker/entrypoint.sh

```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/env bash
 
 set -eo pipefail
+set -x
 
-source ~/.nvm/nvm.sh
 source ~/.cargo/env
 
 sudo dbus-daemon --system
 sudo Xvfb :99 -screen 0 640x480x8 -nolisten tcp &
 
-sleep 1
+sleep 2
 
-video_compositor
+exec "$MEMBRANE_VIDEO_COMPOSITOR_MAIN_EXECUTABLE_PATH"
```

Comment:
exec to make sure signals sent to the parent process are passed to the compositor process